.github/workflows/build_universal_on_forks.yml

```diff
@@ -0,0 +1,36 @@
+---
+# This workflow is required to experiment with container images on forks
+# it will NEVER run on aristanetworks/avd and requires following env variables:
+#     DEBUG_CONTAINER_ACTIONS = true
+# The workflow supposed to be the exact replica of container_build_universal.yml in a stable state
+# but diff is allowed to support a longer experimental setup across different forks
+name: Build universal on forks
+
+"on":
+  push:
+    branches: [ '**' ]
+    paths:
+      - containers/universal/**
+      - .github/workflows/container_build_template.yml
+      - .github/workflows/build_universal_on_forks.yml
+      - ansible_collections/arista/avd/**
+  workflow_dispatch:
+    branches: [ '**' ]
+  release:
+    types: [published]
+
+jobs:
+  build_universal_container:
+    if: github.repository != 'aristanetworks/avd' && ${{ vars.DEBUG_CONTAINER_ACTIONS }} == "true"
```

Comment:
where will these containers be uploaded to? It should be the PR repo and not ours.

Reply:
@ClausHolbechArista The image will be always uploaded to `ghcr.io/${{ github.repository }}/${{ inputs.container_name }}`. As there is nothing hardcoded here - the image will be pushed to the fork repo if initiated on fork branch or to the main repo if initiated on merge to devel / release.